packages/amplify-nodejs-function-template-provider/resources/lambda/appsync-todo/iam/package.json.ejs

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "<%= props.functionName.toLowerCase() %>",
+  "type": "module",
+  "version": "2.0.0",
+  "description": "Lambda function generated by Amplify",
+  "main": "index.js",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@aws-crypto/sha256-js": "^2.0.1",
```

Comment:
are these dependencies available in the NodeJS lambda runtime by default?

Reply:
no we need to add them.